src/workspace/qml/MuseScore/Workspace/WorkspacesDialog.qml

```diff
@@ -86,7 +86,7 @@ StyledDialogView {
         WorkspacesView {
             id: view
             Layout.fillWidth: true
-            Layout.fillHeight: true
+            Layout.preferredHeight: root.height - 190
```

Comment:
@octopols 
This is not the right solution, it's bad to use magic numbers, it's not clear why this number, why not 200 :)

Positioning on the delegate works through positionViewAtIndex. The problem is that positioning doesn't correctly calculate delegate heights.
This is how qml works, need to give time to render the view and only then position on the selected delegate.

You need to call positioning delayed:
` Qt.callLater(root.positionViewAtIndex, selectedWorkspace.index, ListView.Contain)
`

Please prepare a PR

Reply:
![qt_callLater](https://user-images.githubusercontent.com/90182257/219128501-14d01c51-0d73-4646-a131-328cbe311398.png)

Should I call it like this?

Reply:
yes

Reply:
I called it like in the image and reverted the changes in WorkspacesDialog.qml but `positionViewAtIndex` still scrolls it behind the WorkspaceBottomPannel somehow.
![2](https://user-images.githubusercontent.com/90182257/219131690-d60060c6-9416-447c-a44a-aaccada4f40a.png)

When I first tried to solve this problem I scrolled to an index bellow the index which was selected using `root.positionViewAtIndex, selectedWorkspace.index + 1, ListView.Contain` 
![3](https://user-images.githubusercontent.com/90182257/219132591-b4eb7bc6-f996-4ea8-a396-f9766a89e358.png)
But this breaks when the last index is selected. 
I tried using `ListView.Center`, `positionViewAtEnd()` still no help, even tried enclosing them inside `onCountChanged` but it still scrolled behind the bottom pannel.

Reply:
> ![qt_callLater](https://user-images.githubusercontent.com/90182257/219128501-14d01c51-0d73-4646-a131-328cbe311398.png)
> 
> Should I call it like this?

I had clip enabled when I tested this, nothing else was changed